config/config.go

```diff
@@ -56,6 +56,7 @@ var (
 // Config contains configuration options.
 type Config struct {
 	Host             string          `toml:"host" json:"host"`
+	Labels           string          `toml:"label" json:"label"`
```

Comment:
Is it better not to put the labels here? Could you please make it consistent with isolation read by engine type as a session/global variable?

Reply:
So we need to set labels explicit for every session?

Reply:
We can set `@@global.tidb_isolation_read_labels=XXX`

Reply:
IMHO, we'd better use the variable like `tikv_isolation_read_labels` to avoid the complex internal influence between tikv and tiflash.

Reply:
We already have session var indicating do follower read or not. Too many session var will make it not easy to use. And, handling labels in region cache don't influence plan logic if engine is tiflash.

Reply:
If your design only aims at followers read by labels. What about change the config name like `followerReadLabels`?

Reply:
It's changed to `LocationLabels` because I notice that there are also other labels. Thank you!